guides/plugins/plugins/checkout/document/add-custom-document-type.md

```diff
@@ -204,7 +358,7 @@ This could be it already. The [base.html.twig](https://github.com/shopware/platf
 
 ## Adding a number range
 
-You're almost done here. You've got a new document type in the database, a generator for your new document type and it even uses a custom template. However, you also need to add a new number range for your documents, otherwise a new number for your documents wouldn't be generated.
+You're almost done here. You've got a new document type in the database, a renderer for your new document type, and it even uses a custom template. However, you also need to add a new number range for your documents, otherwise a new number for your documents wouldn't be generated.
```

Comment:
```suggestion
You are almost done here. You have a new document type in the database, a renderer for your new document type, and it even uses a custom template. However, you also need to add a new number range for your documents; otherwise, a new number wouldn't be generated for your documents.
```